schematic/schemas/data_model_relationships.py

```diff
@@ -214,6 +228,35 @@ def define_required_csv_headers(self) -> list:
 
         return required_headers
 
+    def define_optional_csv_headers(self) -> list[str]:
+        """Retrieves the optional CSV headers.
+
+        Raises:
+            ValueError: When 'required_header' isn't a key in the header dictionary
+            ValueError: When 'csv_header' isn't a key in the header dictionary
+
+        Returns:
+            list[Optional[str]]: A list of optional csv headers the data model has
+        """
+        headers = []
+        for header_name, header_dict in self.relationships_dictionary.items():
+            if "required_header" not in header_dict:
```

Comment:
Should all of these hard coded strings be extracted out to a constants file? If everything is referenced in a single file, and imported, we will know where everything is explicitly being used without needing to do a repo wide text search - You would be able to see where the constant references are being used via vscode.